docs/config.toml

```diff
@@ -229,13 +235,25 @@ defaultContentLanguage = "en"
     identifier = "wiring"
     weight = 303
     parent = "authoring-bundles"
+  [[menu.main]]
+    name = "Distributing Bundles"
+    url = "/distributing-bundles"
+    identifier = "distributing-bundles"
+    weight = 304
+    parent = "authoring-bundles"
 
 [[menu.main]]
   name = "Examining Bundles"
   identifier = "examining-bundles"
   url = "/examining-bundles"
   weight = 325
 
+[[menu.main]]
+  name = "Copying Bundles"
```

Comment:
What if we made a new top level called "Distributing Bundles" immediately below Authoring Bundles, instead of a child. And then the current "/distributing-bundles" can be the "Overview" (like the other top level groupings have. And Copying Bundles can be a child of this new top level.